CONTRIBUTING.md

```diff
@@ -31,54 +31,52 @@ To contribute to this project, it is recommended that you follow these steps:
 6. Send a GitHub Pull Request to the main repository's master branch. GitHub
    Pull Requests are the expected method of code collaboration on this project.
 
-# Testing
+## Testing
 
-Before running any tests, a supported version of Python must be installed.
-Version 3.5 is preferred, as that version is the most likely to be problematic.
-Installing multiple supported python versions is even better.
+Before running any tests, you must have a supported version of Python (3.8+) and [Poetry](https://python-poetry.org/docs/) installed locally.
 
 It is also helpful to install SystemLink Server and configure the NI Web Server
 to run on localhost.
 
 To install all development dependencies required:
+
 ```
-$ pip install -r .\requirements.txt
+poetry install
 ```
 
-Use the `tox` package for running all tests. To get set up:
+To run commands and scripts, spawn a shell within the virtual environment managed by Poetry:
```

Comment:
Or use `poetry run ...`. I'm not sure which one is preferable for documentation.

Reply:
I documented using `poetry run`